client/src/components/companyDescription.jsx

```diff
@@ -17,7 +18,7 @@ class About extends React.Component {
     for (let i = 0; i < 6; i++) {
```

Comment:
What happens if the sentences end in something other than a period? A question mark, or an exclamation point, for example...

Reply:
fixed